src/Grpc.AspNetCore.Server.ClientFactory/GrpcContextPropagationOptions.cs

```diff
@@ -0,0 +1,34 @@
+﻿#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+namespace Grpc.AspNetCore.ClientFactory
+{
+    /// <summary>
+    /// Options used to configure gRPC call context propagation.
+    /// </summary>
+    public class GrpcContextPropagationOptions
+    {
+        /// <summary>
+        /// Gets or sets a value indicating whether to ignore context not found errors when the client
+        /// attempts to propagate the call context. This can happen when the client is used outside
+        /// the context of a server gRPC call.
+        /// Default value is <c>false</c>.
+        /// </summary>
+        public bool IgnoreContextNotFoundError { get; set; }
```

Comment:
Triage consensus: We're ok with this, or removing the `Error` suffix

Reply:
```suggestion
        public bool ThrowIfContextNotFound { get; set; } = true;
```

Another option. Nobody is to Z E S T Y about this though.